contracts/events/EventFactory.sol

```diff
@@ -10,7 +10,7 @@ contract EventFactory {
 
     // Events
     event TopicCreated(address indexed _topicAddress, address indexed _creator, address indexed _oracle,
-        bytes32[10] _name, bytes32[10] _resultNames, uint256 _bettingEndBlock, uint256 _resultSettingEndBlock);
+        bytes32[10] _name, bytes32[10] _resultNames);
 
```

Comment:
with this change retrieval of _bettingEndBlock and _resultSettingEndBlock need to done from Oracles?